<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QDialog" name="EditWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_4">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>193</width>
             <height>50</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>Логин</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_26">
                <property name="text">
                 <string>Пароль</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QLineEdit" name="lineEdit_username_patient"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_password_patient"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_8">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>225</width>
             <height>182</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item>
               <widget class="QLabel" name="label_27">
                <property name="text">
                 <string>Имя</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_28">
                <property name="text">
                 <string>Фамилия</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_29">
                <property name="text">
                 <string>Отчество</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_30">
                <property name="text">
                 <string>Дата рождения</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_31">
                <property name="text">
                 <string>Личный номер</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_32">
                <property name="text">
                 <string>Домашний адрес</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Доктор</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_16">
              <item>
               <widget class="QLineEdit" name="lineEdit_patient_name"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_patient_surname"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_patient_patronymic"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <widget class="QDateEdit" name="dateEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_patient_cipher"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_patient_HomeAddress"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_patient_attendDoctor"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>128</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_create_patient">
           <property name="text">
            <string>Создать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>193</width>
             <height>50</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Логин</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Пароль</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLineEdit" name="lineEdit_doctor_username"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_doctor_password"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>264</width>
             <height>136</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Имя</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Фамилия</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Отчество</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Дата поступления</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Специальность</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLineEdit" name="lineEdit_doctor_name"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_doctor_surname"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_doctor_patronymic"/>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEdit_doctor_enrollmentDate"/>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_specialities"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>128</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_create_doctor">
           <property name="text">
            <string>Создать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="create_registrator">
           <property name="text">
            <string>Создать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_9">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>193</width>
             <height>50</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Логин</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Пароль</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_username_registrator"/>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_pasword_registrator"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_10">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>198</width>
             <height>76</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_17">
              <item>
               <widget class="QLabel" name="label_33">
                <property name="text">
                 <string>Имя</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_34">
                <property name="text">
                 <string>Фамилия</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_35">
                <property name="text">
                 <string>Отчество</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item>
               <widget class="QLineEdit" name="lineEdit_name_registrator"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_surname_registrator"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_patronymic_registrator"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
